self_test/faceted_surface_vtu_output/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# ------------------------------------------------------------------------------
+list(APPEND CMAKE_MESSAGE_INDENT " ")
+message(VERBOSE "Entered faceted_surface_vtu_output subdirectory")
+
+cmake_minimum_required(VERSION 3.22 FATAL_ERROR)
+project(faceted_surface_vtu_output C CXX Fortran)
+if(NOT oomphlib_FOUND)
+  find_package(oomphlib CONFIG REQUIRED PATHS "../../install")
+endif()
+include(CTest)
+
+oomph_add_executable(
+  NAME faceted_surface_vtu_output_tester
+  SOURCES faceted_surface_vtu_output_tester.cc
+  LIBRARIES oomph::poisson oomph::generic)
+
+message(VERBOSE "Leaving faceted_surface_vtu_output subdirectory")
+# ------------------------------------------------------------------------------
```

Comment:
I think you've mistakenly added this file; the main `oomph-lib` repository doesn't use CMake files (yet!)

Reply:
I've added a commit (https://github.com/oomph-lib/oomph-lib/pull/98/commits/d3231c530db11245c6ce921654fb0f9d44117ed6) that removes this CMakeLists.txt file. I can rebase and completely remove this mistake from the history if you prefer.